README.md

```diff
@@ -31,6 +31,8 @@ Build
 
 ### Requirements
 
+On some systems (e.g. Termux, Alpine Linux): `findutils` `gawk` `zip` `wget` need to be manually installed. Busybox won't work.
```

Comment:
Termux is not a system itself, better to use "environment".
Android doesn't ship Busybox but Toybox, so I don't think the Busybox comment is really useful as it is worded and I find it redundant. Could you remove it?

Reply:
Termux does ship Busybox